pilot/pkg/networking/v1alpha3/cluster.go

```diff
@@ -332,3 +312,38 @@ func setUpstreamProtocol(cluster *v2.Cluster, port *model.Port) {
 		}
 	}
 }
+
+func newCluster(env model.Environment, name string, discoveryType v2.Cluster_DiscoveryType,
```

Comment:
nit in most other places in istio we use `buildXXXX` when building envoy XDS configs